packages/backend/src/controllers/organizationController.ts

```diff
@@ -174,6 +175,41 @@ export class OrganizationController extends Controller {
         };
     }
 
+    /**
+     * Get the membership profile for a user in the current user's organization
+     * @param req express request
+     * @param body the userUuid or email of the user to get. Either userUuid or email must be provided.
+     */
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @Post('/user')
+    @OperationId('ListOrganizationMembers')
+    async getOrganizationMember(
+        @Request() req: express.Request,
+        @Body() body: OrganizationMemberProfileGet,
+    ): Promise<ApiOrganizationMemberProfile> {
```

Comment:
We can also separate the methods to get an user's profile by UUID and email. Which do you prefer?